packages/@ourworldindata/components/src/styles/variables.scss

```diff
@@ -153,6 +153,7 @@ $zindex-popover: 100;
 $zindex-search-overlay: 100;
 $zindex-cookie-notice: 110;
 $zindex-site-header: 120;
+$zindex-extended-header: $zindex-site-header - 1; // ensures search overlays topic-page headers and ToC does not
```

Comment:
I actually meant to move that var defintion above the 120 line (since 119 < 120). That's a very small thing, but I think it'll help with parsing and readability.

Reply:
I wasn't sure, but I can't do that precisely because I preferred to tie the variables together, ie, if someone changes `$zindex-site-header` nothing breaks, even if the two lines diverge in the code.

I suppose one option would be to swap:

```
$zindex-extended-header: 120;
$zindex-site-header: $zindex-extended-header + 1; 
```

Reply:
I'm also happy with 

```
$zindex-extended-header: 119;
$zindex-site-header: 120; 
```

then get it merged given the rethink incoming.

Reply:
no you're right, computing one from the other tells a better story. I was just not thinking straight - using a var before declaring it 🤦‍♂️. It's all good as it is now, I'll go ahead and merge in a bit. Thanks!

Reply:
No problem :-)